modules/core/core-api/src/main/java/com/enonic/xp/query/aggregation/TermsAggregationQuery.java

```diff
@@ -103,7 +103,7 @@ public Builder size( final Integer size )
             return this;
         }
 
-        public Builder minDoccount( final long minDocCount )
+        public Builder minDocCount( final Long minDocCount )
```

Comment:
it is impossible to change method name and arg types in API

Reply:
Done